app/models/withdrawal.rb

```diff
@@ -11,6 +11,8 @@ class Withdrawal < ActiveRecord::Base
 
   after_create :decrease_user_balance
 
+  scope :by_interval, ->(interval) { where('(DATE(?) - DATE(withdrawals.created_at)) < ?', Time.current, interval) }
```

Comment:
Let's add spec as well

Reply:
It is testing user_spec.rb (non_paid_clients)